settings.php

```diff
@@ -152,6 +152,11 @@
                                                     $recentactivitydesc,
                                                     '1'));
 
+        $settings->add(new admin_setting_configtext('tool_crawler/crawl_task',
```

Comment:
see above, name for this 'crawl_task' isn't useful

Reply:
Have changed the name to what it is: max workers